lizrd/support/logging.py

```diff
@@ -342,12 +346,15 @@ def report_scalar(
         value: float,
         iteration: int,
         series: Optional[str] = None,
-        processed_token_scale: bool = False,
+        processed_tokens_so_far: Optional[int] = None,
     ):
         path = self._make_path(title, series)
         wandb.log({path: value, "train/step": iteration})
         auxiliary_metrics = self.get_auxiliary_metrics(
-            title, value, iteration, token_scale=processed_token_scale
+            title,
+            value,
+            iteration,
+            processed_tokens_so_far=processed_tokens_so_far,
```

Comment:
We should delete token_scale / processed_tokens_so_far as it is no longer relevant